README.md

```diff
@@ -1,14 +1,52 @@
-### shared-latent-space
-Shared Latent Space VAE's
+# The Motivation of Shared Latent Space Variational Auto Encoders
+## In Biology
+Shared Latent Space VAE's find relationships between two different domains and allow for transformations between the two. This has especially useful implications in biology, where you may be able to have predictions of how certain tests will look based on ones you already possess. 
```

Comment:
Maybe good to start with one or two sentences about what VAEs are in general (with appropriate citations). Then, what does a shared latent space VAE do that single domain VAEs don't?

Reply:
I am also not sure what you mean by the second sentence. I think the useful applications in biology bit deal mostly with data-type integration. The ability to leverage different data-types in terms of different "views" on the same biological problem. I think the shared latent space angle is also beneficial because of the cycles - can potentially transition biology across domains.

Reply:
Also, each sentence belongs on its own line in markdown files.